content/en/docs/contribute/suggesting-improvements.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Suggesting content improvements
+slug: suggest-improvements
+content_template: templates/concept
+weight: 10
+card:
+  name: contribute
+  weight: 20
+---
+
+{{% capture overview %}}
+
+The easiest way to contribute to Kubernetes documentation file is to an issue about existing content,
+or a request for new content. All you need is a [GitHub account](https://github.com/join) and a web browser.
+
+All new work on Kubernetes documentation begins with an issue in GitHub. Kubernetes contributors
+then review, categorize and tag issues as needed. You or another member
+of the Kubernetes community begin work on the issue an open a pull request with changes,
+and the issue is resolved.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Suggesting improvements to existing content
```

Comment:
nit: I am not sure the title and headings work together
The title is `Suggestion content improvements`. Then the sub headings include,
`... improvements in existing content`
`... new content`